vividus-plugin-mobile-app/src/main/resources/spring.xml

```diff
@@ -100,11 +100,14 @@
             </bean>
         </property>
         <property name="shootingStrategy" value="${mobile.screenshot.strategy}" />
+        <property name="downscale" value="${mobile.screenshot.downscale}" />
     </bean>
 
-    <bean id="mobileAppWebDriverManager" class="org.vividus.selenium.mobileapp.MobileAppWebDriverManager" parent="genericWebDriverManager"/>
+    <bean id="mobileAppWebDriverManager" class="org.vividus.selenium.mobileapp.MobileAppWebDriverManager" parent="genericWebDriverManager" />
 
-    <bean id="coordsProvider" class="org.vividus.selenium.mobileapp.screenshot.NativeHeaderAwareCoordsProvider" />
+    <bean id="coordsProvider" class="org.vividus.selenium.mobileapp.screenshot.DprAwareCoordsProvider">
```

Comment:
it seems `NativeHeaderAwareCoordsProvider` is not going to be used explicitly, so `DprAwareCoordsProvider` and `NativeHeaderAwareCoordsProvider` can be merged into a new `MobileAppCoordsProvider`